docs/examples/haproxy/config.toml

```diff
@@ -3,8 +3,9 @@ dockerURL = "unix:///var/run/docker.sock"
 
 [[extensions]]
 name = "haproxy"
-configPath = "/usr/local/etc/haproxy.cfg"
-pidPath = "/etc/haproxy.pid"
+configPath = "/usr/local/etc/haproxy/haproxy.cfg"
```

Comment:
Since we recommend the official Docker hub HAProxy image and we run in
containers we should use what that image uses. I'll take a look to confirm.
Thanks!
On Apr 6, 2016 17:27, "Everett Toews" <notifications@github.com> wrote:

> In docs/examples/haproxy/config.toml
> <https://github.com/ehazlett/interlock/pull/113#discussion_r58785149>:
>
> > @@ -3,8 +3,9 @@ dockerURL = "unix:///var/run/docker.sock"
> >
> >  [[extensions]]
> >  name = "haproxy"
> > -configPath = "/usr/local/etc/haproxy.cfg"
> > -pidPath = "/etc/haproxy.pid"
> > +configPath = "/usr/local/etc/haproxy/haproxy.cfg"
>
> Ya. haproxy.org isn't too helpful
> <https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=site:www.haproxy.org+%22haproxy.cfg%22>.
> I think it mostly comes down where distros package it, seems like Ubuntu
> <https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=ubuntu%20haproxy.cfg>
> and CentOS
> <https://www.google.com/webhp?sourceid=chrome-instant&ion=1&espv=2&ie=UTF-8#q=centos%20haproxy.cfg>
> favour /etc/haproxy/haproxy.cfg.
>
> That's always been my personal preference too (e.g.
> /etc/appname/appname.conf, /etc/appname/appname.yml, etc.). But honestly I
> don't feel strongly about it here. I'm fine changing it back to
> /usr/local/etc/haproxy.cfg if that's your preference.
>
> —
> You are receiving this because you commented.
> Reply to this email directly or view it on GitHub
> <https://github.com/ehazlett/interlock/pull/113/files/d694e612ab69aa2468f830b5a20828a20ba14d91#r58785149>
>